lnwallet/mock.go

```diff
@@ -0,0 +1,208 @@
+package lnwallet
```

Comment:
ideally mocks should be in files ending with `_test.go` to avoid compiler checking them during non-testing.